node_helper.js

```diff
@@ -7,9 +7,9 @@ module.exports = NodeHelper.create({
       this.sendSocketNotification(
         'SET_LANGUAGE',
         JSON.stringify(
-          locale[payload],
+          Object.assign({}, { language: payload }, locale[payload]),
```

Comment:
You can add the parser if you want. I apparently didn't set it up correctly 😁 This solution is also fine!

Reply:
Actually I resorted to this because eslint does not support object spread bcs it is still considered experimental. it passes the check now :)